Bookish/Views/Book/EditBook.cshtml

```diff
@@ -0,0 +1,28 @@
+﻿@{
+    ViewData["Title"] = "Edit Book";
+}
+@model Bookish.Models.Book
+<div class="jumbotron">
+    <p class="lead">Input a book with information below</p>
+</div>
+
+<div class="row">
+    <div class="col-md-12">
+        @using (Html.BeginForm("BookEdited", "Book", FormMethod.Get))
```

Comment:
As I commented earlier, I don't think this is a GET method - you're submitting data